style.css

```diff
@@ -549,6 +550,69 @@ body > footer {
 	}
 }
 
+@media (max-width: 1000px){
+	body {
+		grid-template:
+			'header header' auto
+			'curve preview' auto
+			'curve library' auto / 300px 1fr;
+	}
+
+	header {
+		text-align: center;
+	}
+
+	header > h1 {
+		display: flex;
+		flex-wrap: wrap;
+		justify-content: center;
+		align-items: center;
+	}
+}
+
+@media (max-width: 850px){
+	body {
+		grid-template:
+			'footer' auto
+			'header' auto
+			'curve' auto
+			'preview' auto
+			'library' auto / auto;
+		padding: 0 5% 1em;
+	}
+
+	body > footer {
+		padding: .5em 0;
+		position: static;
```

Comment:
In general, instead of resetting values to their initial values in a MQ, it's often a better practice to use a MQ to override only when needed (related to the "mobile first" design pattern).